ldi-orchestrator/ldio-connectors/ldio-amqp/src/test/java/be/vlaanderen/informatievlaanderen/ldes/ldio/AmqpInIntegrationTestSteps.java

```diff
@@ -117,11 +117,11 @@ public void theResultValueWillContainTheModel() {
 
 	@When("I pause the pipeline")
 	public void pausePipeline() {
-		ldioInput.updateStatus(HALTED);
+		ldioInput.updateStatus(HALT);
 	}
 	@When("I unpause the pipeline")
 	public void unPausePipeline() {
-		ldioInput.updateStatus(RESUMING);
+		ldioInput.updateStatus(RESUME);
 	}
 	@And("The result value will not contain the model")
```

Comment:
Could this be formatted please, as I see there is no whitespace between these methods